src/mds/MDSMap.cc

```diff
@@ -542,7 +544,7 @@ void MDSMap::encode(bufferlist& bl, uint64_t features) const
     return;
   }
 
-  ENCODE_START(5, 4, bl);
```

Comment:
You need to bump the "ev" below from 10 to 11 instead of modifying this version (the mdsmap encoding is weird!)